raster-foundry/raster-foundry
# Recalculate and update project extent after scene deletion

## Overview

This PR adds project extent recalculation on backend after deleting scenes. It also refetches and updates map extent after scene deletion on frontend.

### Checklist


## Testing Instructions

 * Go to a project with scenes distant from each other
 * Delete scenes -- the map should fit to the new extent of the project
 * Go to project list page -- the map should use the new project extent too

Closes https://github.com/raster-foundry/raster-foundry/issues/3286
